backend/tests/conftest.py

```diff
@@ -475,8 +481,6 @@ async def car_person_schema_unregistered(db: InfrahubDatabase, node_group_schema
                         "name": "owner",
                         "label": "Commander of Car",
                         "peer": "TestPerson",
-                        "optional": False,
-                        "kind": "Parent",
```

Comment:
please can you provide more information about this change ? why is it related to this PR ?

Reply:
Same as above - it should not be there